pubspec.yaml

```diff
@@ -17,7 +17,7 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  analyzer: '>=0.36.0 <0.40.0'
+  analyzer: '>=0.36.0 <0.41'
```

Comment:
This needs to be 0.41.0.
I suspect that the failures on Travis are not caused by the async imports removal, but due to the SDK Travis uses for testing being updated.  So an empty PR that just fixes the Travis errors would unblock all the pending PRs that are failing on Travis.